parcels/particleset.py

```diff
@@ -273,20 +273,27 @@ def execute(self, pyfunc=AdvectionRK4, starttime=None, endtime=None, dt=1.,
             if interval < 0:
                 interval *= -1.
                 logger.warning("Negating interval because running in time-forward mode")
-        if endtime < starttime:  # Time-backward mode
+        elif endtime < starttime:  # Time-backward mode
             if dt > 0.:
                 dt *= -1.
                 logger.warning("Negating dt because running in time-backward mode")
             if interval > 0.:
                 interval *= -1.
                 logger.warning("Negating interval because running in time-backward mode")
 
+        if np.allclose(endtime, starttime) or interval == 0 or dt == 0 or runtime == 0:
+            timeleaps = 1
+            dt = 0
+            runtime = 0
+            endtime = starttime
```

Comment:
Add a `logger.warning_once` here that kernels are executed without incrementing time?

Reply:
It is done, but git didn't see it (it's just next line)